starfish/test/dataset_fixtures.py

```diff
@@ -29,9 +29,20 @@ def merfish_stack() -> Stack:
     return s
 
 
+def default_tile_data_provider(hyb: int, ch: int, z: int, height: int, width: int) -> np.ndarray:
```

Comment:
doc each of these new functions please, I imagine devs (me, heh) will want to use these. :)